core/spec/services/spree/cart/remove_line_item_spec.rb

```diff
@@ -16,11 +16,30 @@ module Spree
         expect { execute }.to change { order.line_items.count }.by(-1)
         expect(execute).to be_success
         expect(value).to eq line_item
-        order.reload
         expect(order.amount).to eq 0
       end
     end
 
+    context 'remove line item' do
```

Comment:
The naming isn't perfect here. There already exists a `remove line item` context and by design contexts should be used to group test cases **to run against the same state** so I don't think this file is semantically accurate.